sections/semantics-forms.include

```diff
@@ -6902,6 +6925,10 @@ You cannot submit this form when the field is incorrect.</samp></pre>
      </tr>
     </table>
 
+<h6 id="the-capture-attribute">The <code>capture</code> attribute</h6>
+
+  The <dfn element-attr for="input"><code>capture</code><dfn> attribute is defined in the [[!html-media-capture]] specification. The <code>capture</a> attribute specifies the preferred facing mode for the media capture mechanism.
```

Comment:
mismatched element (opens as `code`, closed as `a`.

Reply:
Rather than defining the attribute in here we should probably refer to the source spec. I will make a PR against the branch to do that.